src/analysis/convex/cone.lean

```diff
@@ -616,4 +616,17 @@ lemma inner_dual_cone_le_inner_dual_cone (h : t ⊆ s) :
 lemma pointed_inner_dual_cone : s.inner_dual_cone.pointed :=
 λ x hx, by rw inner_zero_right
 
+/-- The dual cone of `s` equals the intersection of dual cones of the points in `s`. -/
+lemma inner_dual_cone_eq_Inter_inner_dual_cone_singleton :
+  (s.inner_dual_cone : set H) = ⋂ i : s, (({i} : set H).inner_dual_cone : set H) :=
```

Comment:
I feel like we're missing a much more primivite result here: is this true?
```lean
lemma inner_dual_cone_Union {ι : Type*} (f : ι → set H):
  (⋃ i, f i).inner_dual_cone = ⨅ i, (f i).inner_dual_cone :=
begin
```

Reply:
I was hoping it could be proven by
```
lemma inner_dual_cone_gc :
  galois_connection (λ s : set H, order_dual.to_dual (s.inner_dual_cone))
    (λ x, (coe (order_dual.of_dual x))) :=
λ s, order_dual.rec $ by exact λ c, show c ≤ s.inner_dual_cone ↔ s ⊆ c, begin
  sorry
end

lemma inner_dual_cone_eq_Inter_inner_dual_cone_singleton {ι : Type*} (f : ι → set H):
  (⋃ i, f i).inner_dual_cone = ⨅ i, (f i).inner_dual_cone :=
(@inner_dual_cone_gc H _).l_supr
```
but I don't think the sorry is true. Do you know of a galois connection hiding here somewhere?